.changeset/stale-papayas-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+---
+
+Fixed controller extension name longer than 64 chars error not showing up
```

Comment:
Please add also the `@sap-ux/preview-middleware`, `patch` is sufficient.

Reply:
so like this?
---
'@sap-ux-private/preview-middleware-client': patch
'@sap-ux/preview-middleware': patch
---

Fixed controller extension name longer than 64 chars error not showing up